ember-cli-build.js

```diff
@@ -26,12 +48,12 @@ module.exports = function(/* defaults */) {
     destDir: '/tests'
   });
 
-  var srcJSHint = new JSHint(src);
   var testsJSHint = new JSHint(tests);
+  var srcTSLint = new TSLint(srcTS);
 
-  var allTests = concat(new MergeTrees([tests, srcJSHint, testsJSHint]), {
+  var allTests = concat(new MergeTrees([tests, testsJSHint, srcTSLint]), {
     outputFile: '/tests/tests.js',
-    inputFiles: ['**/*.js'],
+    inputFiles: ['**/*.js', '**/*.ts'],
```

Comment:
FYI - I opened an issue upstream regarding the extension here: https://github.com/kratiahuja/broccoli-tslinter/issues/13